File: 프로그래머스/0/181875. 배열에서 문자열 대소문자 변환하기/배열에서 문자열 대소문자 변환하기.py
def solution(strArr):
    answer = []
    for i in range(len(strArr)):
        if i == 0:
            answer.append(strArr[0].lower())
        elif i % 2 == 0:
            answer.append(strArr[i].lower())
        else:
            answer.append(strArr[i].upper())
    return answer

File: 프로그래머스/0/181875. 배열에서 문자열 대소문자 변환하기/배열에서 문자열 대소문자 변환하기.java
import java.util.*;

class Solution {
    public String[] solution(String[] strArr) {
        String[] answer = new String[strArr.length];
        List<String> al = new ArrayList<>();
        for (int i = 0; i < strArr.length; i++) {
            if (i % 2 == 0 || i == 0) {
                al.add(strArr[i].toLowerCase());
            }
            else al.add(strArr[i].toUpperCase());
        }
        
        
        return al.stream().toArray(String[]::new);
    }
}